Logico/codigo/Locacao.pl

```diff
@@ -0,0 +1,108 @@
+:-use_module(library(csv)).
+:-include('Arquivos.pl').
+
+% Cadastra uma locação ao fim do arquivo Locacoes.csv
+cadastraLocacao(IdFilme, CpfCliente, DataLocacao, Status) :-
```

Comment:
comentando aqui só pra eu lembrar depois de decrementar o estoque do filme

Reply:
Lembra de incrementar quando quando o filme for "devolvido"

Reply:
Lembrar também de settar o status automaticamente para 'em andamento'